projects/bp-gallery/src/components/views/location-curating/LocationManagementDialog.scss

```diff
@@ -0,0 +1,150 @@
+  height: 616.5px;
+}
+
+.location-management-location-name {
+  padding: 0;
+  margin: 0;
+}
+
+.location-management-close-dialog {
+  margin-left: auto;
+  margin-right: 0;
+}
+
+.location-management-dialog-container {
+  display: flex;
+  flex-direction: row;
+  margin-top: -20px;
+  margin-bottom: -20px;
+
+  .location-management-left {
+    padding-right: 8px;
+    .location-management-name-container {
+      display: flex;
+
+      .location-management-accept-location {
+        margin: auto auto auto 8px;
+      }
+
+      .location-management-edit-location {
+        margin: auto 0px auto auto;
+      }
+
+      .MuiButtonBase-root {
+        padding: 0;
+        margin-left: auto;
+      }
+    }
+
+    .location-management-location-path {
+      width: 505px;
+    }
+
+    .location-management-left-content {
+      height: 510px;
+      overflow: scroll;
+      padding-right: 16px;
+
+      .location-management-synonyms {
+        max-width: 450px;
+        margin-bottom: 4px;
+        display: flex;
+        flex-direction: row;
+        overflow-wrap: normal;
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+
+        .location-management-synonyms-grid {
+          margin-bottom: auto;
+          margin-top: 0;
+          overflow: scroll;
+
+          .location-management-synonym {
+            margin: 1px 2px 1px 2px;
+          }
+        }
+      }
+    }
+  }
+
+  .location-management-right {
+    padding-left: 8px;
+    max-width: 400px;
+    .location-management-map {
+      width: 400px;
+      height: 406px;
+      background-color: greenyellow;
+    }
+
+    .location-management-actions {
+      .location-management-picture-count {
+        width: 100%;
+        text-align: center;
+        color: $archiveColor;
+        margin-top: 8px;
+      }
+
+      .location-management-show-pictures-button {
+        background-color: $primaryColor;
+        color: #ffffff;
+        margin-top: 0.5rem;
+        width: 100%;
+        padding-inline: 1rem;
+        align-self: center;
+      }
+
+      .location-management-show-button {
+        background-color: $primaryColor;
+        color: #ffffff;
+        margin-top: 0.5rem;
+        width: 100%;
+        padding-inline: 1rem;
+        align-self: center;
+      }
+
+      .location-management-not-show-button {
+        background-color: lightgray;
+        color: #ffffff;
+        margin-top: 0.5rem;
+        width: 100%;
+        padding-inline: 1rem;
+        align-self: center;
+      }
+
+      .location-management-root-button {
+        background-color: $primaryColor;
+        color: #ffffff;
+        margin-top: 0.5rem;
+        width: 100%;
+        padding-inline: 1rem;
+        align-self: center;
+      }
+
+      .location-management-not-root-button {
+        background-color: lightgray;
+        color: #ffffff;
+        margin-top: 0.5rem;
+        width: 100%;
+        padding-inline: 1rem;
+        align-self: center;
+      }
```

Comment:
These have a lot of shared code, maybe put the common code inside a class and change the special code through inner classes? This would also benefit the tsx code